src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -171,6 +171,31 @@ StorageBond1Interface1=em2
 StorageBond1Interface2=p2p2
 StorageBondInterfaceOptions=mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
 
+[Dell NFV Settings]
+
+# OVS-DPDK Settings
+# ovs_dpdk_enable supports three values
+# ovs_dpdk_enable=None: disables OVS-DPDK
+# ovs_dpdk_enable=Tenant-networks-only: Enables OVS-DPDK in Mode 1.
+# ovs_dpdk_enable=Tenant-and-External-networks: Enables OVS-DPDK in Mode 2.
+ovs_dpdk_enable=Tenant-networks-only
+
+# Comma separated list of roles on which OVS-DPDK needs to be enabled
+# Only supported value is "Compute".
+ovs_dpdk_role_list=Compute
+
+# Cores distribution policy to be used. Only supported value is "Balanced".
+ovs_dpdk_policy=Balanced
+# Refer to guide for more details about each parameter.
+
+hpg_enable=False
```

Comment:
The hpg_enable through numa_flavor_name_list parameter should be removed from this patch since they are being added under the hpg/numa patch.

Reply:
Agree. Removed.